build/registry/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM quay.io/openshift/origin-operator-registry
 
 COPY manifests /registry
 
+RUN yum -y update
```

Comment:
Oh you can't run this command on this image (this user there doesn't have permissions) I think we can remove the `yum update` from this image

Reply:
quay reports that it found vulnerabilities there, lemme try to use sudo

Reply:
`sudo` is not installed in the image. dropped update of the registry image for now.

Reply:
I think the pod doesn't have the sudo command. you can try to save the user then run `USER root` yum update and return the USER again.

I am not sure if this going to work but you can try